.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+## Intent
+
+> Describe what problem this pull request resolves, or what new feature it
+> introduces.
+>
+> - Link to related GitHub issues, if applicable
+
+Fixes #
+
+## Approach
+
+> Explain how you solved the problem and any tradeoffs.
+>
+> If applicable:
+> - Include guidance for reviewers (rewrites ..., start with ..., commit-by-commit, etc.).
+> - Link to notes and discussions about this work.
+
+## Automated Testing
+
+> Describe the automated testing of this change.
+>
+> - What cases are covered by unit tests?
+> - What cases are covered by integration tests?
```

Comment:
I don't think this is necessary since any changes should be tested.

Reply:
Yeah, sounds good.